plugins/c9.ide.ui/lib_apf.js

```diff
@@ -3726,7 +3726,7 @@ apf.setNodeValue = function(xmlNode, nodeValue) {
     else {
         var oldValue = xmlNode.nodeValue;
         xmlNode.nodeValue = nodeValue === undefined || nodeValue === null ||
-            nodeValue == NaN ? "" : String(nodeValue);
+            isNaN(nodeValue) ? "" : String(nodeValue);
```

Comment:
this breaks the ide because `isNaN("string")` is `true`. I am changing this line to `xmlNode.nodeValue = nodeValue == null ? "" : String(nodeValue);`, otherwise the pull request looks good, thanks!